scripts/package.js

```diff
@@ -239,11 +294,6 @@ export default class Package {
             this.formatError(event.error)
             return this.logger.error(event.error)
 
-          // Encountered an unrecoverable error
-          case 'FATAL':
-            this.formatError(event.error)
-            return this.logger.fatal(event.error)
-
```

Comment:
There is no longer a FATAL event.

https://github.com/rollup/rollup/issues/3336